exercises/vecs/vecs2.rs

```diff
@@ -7,7 +7,6 @@
 //
 // Execute `rustlings hint vecs2` or use the `hint` watch subcommand for a hint.
 
-// I AM NOT DONE
```

Comment:
- [ ] Use [`Iterator::for_each()`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.for_each)

Reply:
changed it